mcstatus/status_response.py

```diff
@@ -65,291 +15,6 @@ class RawJavaResponse(TypedDict):
-        )
-
-
-@dataclass
-class BedrockStatusPlayers(BaseStatusPlayers):
-    """Class for storing information about players on the server."""
-
-
-@dataclass
-class JavaStatusPlayer:
-    """Class with information about a single player."""
-
-    name: str
-    """Name of the player."""
-    id: str
-    """ID of the player (in `UUID <https://en.wikipedia.org/wiki/Universally_unique_identifier>`_ format)."""
-
-    @property
-    def uuid(self) -> str:
-        """Alias to :attr:`.id` field."""
-        return self.id
-
-    @classmethod
-    def build(cls, raw: RawJavaResponsePlayer) -> Self:
-        """Build :class:`JavaStatusPlayer` from raw response :class:`dict`.
-
-        :param raw: Raw response :class:`dict`.
-        :raise ValueError: If the required keys (``name``, ``id``) are not present.
-        :raise TypeError: If the required keys (``name`` - :class:`str`, ``id`` - :class:`str`)
-            are not of the expected type.
-        :return: :class:`JavaStatusPlayer` object.
-        """
-        return cls(name=raw["name"], id=raw["id"])
-
-
-@dataclass
-class BaseStatusVersion(ABC):
-    """A class for storing version information."""
-
-    name: str
-    """The version name, like ``1.19.3``.
-
-    See `Minecraft wiki <https://minecraft.wiki/w/Java_Edition_version_history#Full_release>`__
-    for complete list.
-    """
-    protocol: int
-    """The protocol version, like ``761``.
-
-    See `Minecraft wiki <https://minecraft.wiki/w/Protocol_version#Java_Edition_2>`__.
-    """
-
-
-@dataclass
-class JavaStatusVersion(BaseStatusVersion):
-    """A class for storing version information."""
-
-    @classmethod
-    def build(cls, raw: RawJavaResponseVersion) -> Self:
-        """Build :class:`JavaStatusVersion` from raw response dict.
-
-        :param raw: Raw response :class:`dict`.
-        :raise ValueError: If the required keys (``name``, ``protocol``) are not present.
-        :raise TypeError: If the required keys (``name`` - :class:`str`, ``protocol`` - :class:`int`)
-            are not of the expected type.
-        :return: :class:`JavaStatusVersion` object.
-        """
-        return cls(name=raw["name"], protocol=raw["protocol"])
-
-
-@dataclass
-class BedrockStatusVersion(BaseStatusVersion):
-    """A class for storing version information."""
-
-    name: str
-    """The version name, like ``1.19.60``.
-
-    See `Minecraft wiki <https://minecraft.wiki/w/Bedrock_Edition_version_history#Bedrock_Edition>`__
-    for complete list.
-    """
-    brand: str
-    """``MCPE`` or ``MCEE`` for Education Edition."""
+import warnings
 
-    @property
-    @deprecated(replacement="name", date="2023-12")
-    def version(self) -> str:
-        """
-        .. deprecated:: 11.0.0
-            Will be removed 2023-12, use :attr:`.name` instead.
-        """
-        return self.name
+warnings.warn("`mcstatus.status_response` is deprecated, use `mcstatus.responses` instead", DeprecationWarning, stacklevel=2)
```

Comment:
We could use `utils.deprecated` like this instead of manually calling `warnings.warn`

```suggestion
deprecated("mcstatus.status_response", replacement="mcstatus.responses", date="2023-12", msg="module was renamed")()
```

Reply:
`mcstatus.utils.deprecated` doesn't work with strings:
```
  File "/home/perchun/programming/mcstatus/mcstatus/utils.py", line 156, in decorate
    name = getattr(obj, "__qualname__", obj.__name__)
                                        ^^^^^^^^^^^^
AttributeError: 'str' object has no attribute '__name__'. Did you mean: '__ne__'?
```

Reply:
This is no longer relevant, since #687 was merged